src/NServiceBus.Core/Persistence/Msmq/SubscriptionStorage/MsmqSubscriptionStorageQueue.cs

```diff
@@ -51,6 +52,7 @@ public void TryReceiveById(string messageId)
         {
             try
             {
+                //todo: add failing test before fixing this
```

Comment:
Would there be harm in changing and aligning it? Or is the intention of that question to keep the fix as focused as possible?